pandas/core/array_algos/masked_accumulations.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+from typing import Callable
+
+import numpy as np
+
+from pandas.core.dtypes.common import (
+    is_bool_dtype,
+    is_float_dtype,
+    is_integer_dtype,
+)
+
+"""
```

Comment:
id expect this at the top of the file.  does one of the linters not like that?

Reply:
Moved